draft-ietf-rtcweb-jsep.xml

```diff
@@ -3882,6 +3892,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
           <t>If the m= section proto value indicates use of SCTP:
           <list style="symbols">
+            <t>If an SCTP association exists, and the remote SCTP port
+            has changed, discard the existing SCTP association. This
+            includes the case when the PeerConnection state is
+            "have-remote-pranswer".</t>
 
             <t>If no SCTP association yet exists, prepare to initiate a
```

Comment:
This sentence probably needs to change to indicate that the SCTP association needs to be created at this point if it was just destroyed above.